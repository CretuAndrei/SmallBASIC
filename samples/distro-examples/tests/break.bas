'

SUB EXSUB
  EXIT SUB
  ? "SUB: EXIT SUB FAILED"
END

? "TEST REPEAT"
REPEAT
  EXIT LOOP
  ? "REPEAT: EXIT LOOP FAILED"
UNTIL 1

? "TEST WHILE"
WHILE 1
  EXIT LOOP
  ? "WHILE: EXIT LOOP FAILED"
WEND

? "TEST FOR"
FOR I=1 TO 20
  EXIT FOR
  ? "FOR: EXIT FOR FAILED"
NEXT

? "TEST SUB"
EXSUB
? "DONE"


