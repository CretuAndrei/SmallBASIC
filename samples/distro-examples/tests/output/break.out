TEST REPEAT
TEST WHILE
TEST FOR
TEST SUB
DONE
